checker.js

```diff
@@ -0,0 +1,175 @@
+    }
+    catch (err) {
+        return console.log('\n-> Error trying to copy "package.json"\n'.red);
+    }
+
+    Superagent.get(LIST_MODULES_URL)
+        .end(function (err, res) {
+            if (err) {
+                return console.log('\n-> Error loading modules from webtask.io\n'.red);
+            }
+
+            var package   = JSON.parse(fs.readFileSync(pkg).toString());
+            var libraries = res.body;
+
+            Object.keys(package.dependencies).forEach(function (key) {
+                var wtLib        = libraries.modules.filter(findByName(key)).reverse().pop();
+                var localVersion = readVersion(package.dependencies[key]);
+
+                if (wtLib && wtLib.version !== localVersion) {
+                    if (opt.interactive) {
+                        var value = readline.keyInYN('->'.blue + ' Do you want to modify ' + key.green + ' from '+ ('v' + localVersion).green +' to ' + ('v' + wtLib.version).green + '?');
+
+                        if (value) {
+                            package.dependencies[key] = wtLib.version;
+                        }
+                    } else {
+                        console.log('->'.blue + ' Updating %s from %s to %s', format(key, 25).green, format(localVersion, 8).green, format(wtLib.version, 8).green);
+                        package.dependencies[key] = wtLib.version;
+                    }
+                }
+            });
+
+            try {
+                fs.writeFileSync(pkg, JSON.stringify(package, null, 2));
+                return console.log('\n-> "package.json" updated successfully\n'.green);
+            }
+            catch (err) {
+                console.log(err);
+                return console.log('\n-> Error updating "package.json"\n'.red);
+            }
+        });
+}
+
+function check() {
+    var table = new Table({
+        head: ['', 'Dependency', 'Version', 'Available version'],
+        colWidths: [3, 30, 20, 20],
+        style : {compact : true, head: ['white']}
+    });
+
+    try {
+        fs.statSync(pkg);
+    }
+    catch (err) {
+        return console.log('\n-> "package.json" does not exists\n'.red);
+    }
+
+    Superagent.get(LIST_MODULES_URL)
+        .end(function (err, res) {
+            if (err) {
+                return console.log('\n-> Error loading modules from webtask.io\n'.red);
+            }
+
+            var libraries    = res.body;
+            var package      = JSON.parse(fs.readFileSync(pkg).toString());
+            var dependencies = package.dependencies;
+            var isCompliant  = true;
+
+            if (dependencies) {
+                Object.keys(dependencies).forEach(function (key) {
+                    var localVersion = readVersion(dependencies[key]);
+                    var lib          = libraries.modules.filter(findByNameAndVersion(key, localVersion)).pop();
+
+                    lib = lib || libraries.modules.filter(findByName(key)).pop() || {version: '-'};
```

Comment:
`findByName` can be replaced by `_.matchesProperty('name', key)` See: https://lodash.com/docs#matchesProperty